edenia/antelope-tools
# Chore/minor adjustments to logos 1315

### Minor adjustments in the logos and include some new icons for the sidebar

### What does this PR do?

- Resolve #1315 
- Resolve #1283

### Steps to test

1. Run the project locally
2. Go to the homepage
3. Check the Jungle logo and change in the XPR text
4. Go to any other page
5. Check the network selector and icons in the sidebar

### Screenshots

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/37b79349-d9c5-43f5-8e11-f5fd0438550a)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/10feab9f-dd9e-4738-b357-2d6d7941c577)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/08ba56e2-a713-4d99-9366-941f3b59a9c9)
